### ISSUE TITLE
Remove Vendor files

### ISSUE THREAD

**Issue by OzairP** [openemr/openemr#1601]
Vendor files should not be commited into git. Lock files are already in git and is enough to affirm consistency of dependancies

https://getcomposer.org/doc/faqs/should-i-commit-the-dependencies-in-my-vendor-directory.md

**Comment by bradymiller** [comment]
Hi @OzairP ,

The community is still learning how to best use composer. We decided to populate vendor until:
1. Developers were comfortable with composer
2. We have a bulletproof and streamlined build process when build openemr packages for production

Check out this wiki page tracking the Composer project (as with bower away project, getting somebody interested in working on this, would be awesome):
https://www.open-emr.org/wiki/index.php/Composer

thanks,
-brady

**Comment by bradymiller** [comment]
@robertdown , just checking that you meant to close this issue :)

**Comment by robertdown** [comment]
Yes, intentionally closed

**Comment by bradymiller** [comment]
@robertdown ,
This is something that will be tackled in the future, so prob good to keep it open. Doing it for the next 5.0.2 release would be to early. I could assign this to a 5.0.3 milestone (ie. something to do in future, but not for the next release). Thoughts?
-brady

**Comment by robertdown** [comment]
We would need to have documentation for developers established prior to doing this. I’m ok with a target of 5.0.3. But I thought there was some additional reason we were requiring the vendor files

**Comment by bradymiller** [comment]
Yep, this is definitely more of a long term goal since there are many other things to focus on and this will require developer building and build pipelines etc. Will change it to 5.0.3 which basically means very low priority.

**Comment by OzairP** [comment]
Moving over should not be much of a hassle as long as no one has edited the vendor files. I believe that if PHP developers are smart enough to work on a system like this, I can garentee that they understand how Composer works and they don't need prior documentation to understand it.

Additionally you can setup scripts to automatically `composer install` on git updates.
